lib/approval_request_listener.rb

```diff
@@ -32,8 +32,17 @@ def subscribe_to_approval_updates
   def process_event(topic)
     approval = ApprovalRequest.find_by!(:approval_request_ref => topic.payload["request_id"])
     approval.order_item.update_message("info", "Task update message received with payload: #{topic.payload}")
+    approval.order_item.update_message("info", "Approval #{approval.id} #{topic.payload['decision']}")
```

Comment:
@mkanoor Do you have thoughts on progress messages and/or frequency of them?

Reply:
@syncrou It shouldn't matter since we are reporting whatever we see in the queue, it is just glorified logging that is visual to the user as opposed to trudging through logs. In this case it depends on when Approval sends messages into us, we will record that. I think it might be just doing the bookends, email sent, denied/approved etc.

Reply:
@lindgrenj6 after discussing with @mkanoor we'd like to make sure the following ProgressMessages and Rails logging statements are taken care of:

Approval Request Event approved
Approval Request Denied with the extra message
Log the payload dump to `Rails.logger.info`